src/gpl/src/nesterovBase.cpp

```diff
@@ -1961,8 +1961,8 @@ void NesterovBase::updateGradients(std::vector<FloatPoint>& sumGrads,
   debugPrint(
       log_, GPL, "updateGrad", 1, "DensityPenalty: {:g}", densityPenalty_);
 
-#pragma omp parallel for num_threads(nbc_->getNumThreads()) \
-    reduction(+ : wireLengthGradSum_, densityGradSum_, gradSum)
+  //#pragma omp parallel for num_threads(nbc_->getNumThreads()) reduction(+ :
```

Comment:
Remove commented code if there are no plans to restore.

Reply:
I am trying to use doubles for additions as suggested by @maliberty. Although it is not working, as least the way I tried so far.

I thought of leaving as a command to try and restore the parallelism in the future.

Reply:
So, add a line mentioning that the goal is to restore it. Having only the commented code might lead to confusion for readers.

Reply:
Have you tried sorting the list floats and then doing a parallel tree reduce?